specs/electra/validator.md

```diff
@@ -148,6 +179,25 @@ def prepare_execution_payload(state: BeaconState,
     )
 ```
 
+#### Execution Requests
+
+*[New in Electra]*
+
+1. The execution payload is obtained from the execution engine as defined above using `payload_id`. The response also includes a `execution_requests` entry containing a list of bytes. Each element on the list corresponds to one ssz list of requests as defined
+in [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685). The index of each element in the array determines the type of request.
+2. Set `block.body.execution_requests = get_execution_requests(execution_requests)`, where:
+
+```python
+def get_execution_requests(execution_requests: list[bytes]) -> ExecutionRequests:
+    requests = ExecutionRequests()
+
+    requests.deposits = deserialize(execution_requests[0], DepositRequest)
+    requests.withdrawals = deserialize(execution_requests[1], WithdrawalRequest)
+    requests.consolidations = deserialize(execution_requests[2], ConsolidationRequest)
+
+    return requests
```

Comment:
it can be:
```python
return ExecutionRequests(deposits, withdrawals, consolidations)
```
rather than creating an empty object and then doing assignment